python/lsst/cp/verify/verifyBias.py

```diff
@@ -37,6 +37,7 @@ def setDefaults(self):
                                   'NOISE': 'STDEVCLIP', }
 
         self.crImageStatKeywords = {'CR_NOISE': 'STDEV', }  # noqa F841
+        self.metadataStatKeywords = {'RESIDUAL STDEV': 'AMP', }  # noqa F841
```

Comment:
Other keys, such as `CR_NOISE`, have `_` in between words. Would it be better to also add an underscore here to be consistent?

Reply:
This comes from a choice in `ip_isr`.  I wanted to keep changes out of that.  I had thought about moving this from the task metadata to the exposure metadata, and then making the metadata stats code look there instead of the task metadata (removing that connection).  However, this wouldn't be available for previously processed data, and so there'd always be two versions.

This was a design decision, and I'm still not sure I made the right one.